src/tyr/serializers.cc

```diff
@@ -209,6 +234,42 @@ json::ArrayPtr waypoints(const valhalla::Trip& trip) {
   return waypoints;
 }
 
+json::ArrayPtr via_waypoints(valhalla::Options options, const std::vector<PointLL>& shape) {
```

Comment:
Could you add a doc-string describing what this function does? And also for the `via_waypoint` above?